src/v/features/feature_table.h

```diff
@@ -146,6 +147,12 @@ constexpr static std::array feature_schema{
     feature::transaction_ga,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{6},
```

Comment:
Version 6 is the 22.2.x backport of transactions, right?  For stuff that is new in 22.3, you need to increment the cluster version.